### ISSUE TITLE
Navigation feature

### ISSUE THREAD

**Issue by Cocola** [styledown/styledown#12]
Hi,
It's not an issue but a feature request.
Are you going to develop a navigation through all the styleguides components ?
Like an aside navbar with link pointing to an anchor :)

In a more general way, are you going to devellop/maintain new features in the future ?

Thanks

**Comment by jlmakes** [comment]
+1

**Comment by danmv** [comment]
+1

**Comment by comoris** [comment]
+1

**Comment by rstacruz** [comment]
Yep! It ain't much yet but this is something being worked on for the v2 rewrite. The idea is to have something like the sidebar of http://rscss.io.

![image](https://cloud.githubusercontent.com/assets/74385/15405659/64d18bda-1e35-11e6-83f0-c0d590199287.png)

**Comment by olthof** [comment]
+1

Love this tool, only thing really missing is a menu / navigation. Thanks for creating this @rstacruz !

**Comment by rstacruz** [comment]
Yeah, the fatal flaw of Styledown v1 is that it can't support multiple files at all. By extension, it can't support navigation. The v2 branch can do this, though—but with no CSS comment parsing support :-)

**Comment by planktonWD** [comment]
I use this: https://github.com/drakeh/styledown-skins it has a navigation and works great if using inline jade commenting in your SCSS files. I love styledown @rstacruz !

**Comment by jermartin77** [comment]
As a quick fix solution I've build out a javascript driven anchor navigation that just indexes all the header tags with ids on the page and nests them. Its still pretty early on but it works. 

![image](https://cloud.githubusercontent.com/assets/1685645/20804539/899b1746-b7b0-11e6-9370-f322ba8dd8f8.png)

**Comment by olthof** [comment]
Do you have an example / code we can use anywhere @jermartin77 ? Does it automatically pick up the id's and build the navigation or is it hardcoded?

**Comment by jermartin77** [comment]
I'm not an expert at scripting the actual styledown build script but at
this point I'm just firing off a javascript loop that builds navigation
based on the existing h2 and h3 elements on the page. I don't have a live
example but I can happily send you the script. Ideally it would just be
part of the compile.
